### DISCUSSION TITLE
How to decelerate slowly

### DISCUSSION THREAD

**Discussion by Xylar52** [pybricks/support#1013] [Q&A] [answered]
i can't seem to get the robot Spike Prime to brake instead of just stopping hard. I used the example code to see the difference between motor.stop and motor.brake and both just makes the robot stop instantly. And even if i succeded to make the robot brake, i don't how to make both motor brake at the same time while using drivebase, because drivebase doesn't have a brake function and it's stop function just makes the robot stop instantly

```
from pybricks.pupdevices import Motor
from pybricks.parameters import Port
from pybricks.tools import wait

# Initialize a motor on port A.
example_motor = Motor(Port.A)

# Run at 500 deg/s and then stop by coasting.
example_motor.run(500)
wait(1500)
example_motor.stop()
wait(1500)

# Run at 500 deg/s and then stop by braking.
example_motor.run(500)
wait(1500)
example_motor.brake()
wait(1500)
```

**Comment by laurensvalk** [comment, marked as answer]
`brake` makes it stop even faster than `stop`, which just coasts the motor. These motors stop almost instantly because of friction.

It sounds like you want slower deceleration?

You can adjust the motor acceleration and deceleration as follows:

```python
example_motor.control.limits(acceleration=(2000, 500))
example_motor.run_angle(1000, 360*4)
```

In this case, it accelerates at 2000 deg/s^2 and decelerates slowly at 500 deg/s^2.

If you do this for both motors and then create the drivebase, it will use those settings as well.

**Reply by Xylar52** [reply]
oh i didn't even know you could adjust motor deceleration, you learned me something. Thanks

**Reply by laurensvalk** [reply]
It's a fairly new feature :)
